examples/jsm/webxr/XRHandModelFactory.d.ts

```diff
@@ -0,0 +1,28 @@
+import {
+	Group,
+	Object3D,
+} from '../../../src/Three';
+
+import { GLTFLoader } from '../loaders/GLTFLoader';
+
+export class XRHandModel extends Object3D {
+
+	constructor();
+
+	motionController: any;
+
+}
+
+export class XRHandModelFactory {
+
+	constructor( gltfLoader?: GLTFLoader );
```

Comment:
This ctor of `XRHandModelFactory` has no parameters. 

It also only `path` as a property.

Reply:
Thanks. Pardon my oversight.